internal/containerizedengine/update.go

```diff
@@ -24,15 +22,34 @@ import (
 
 // ActivateEngine will switch the image from the CE to EE image
 func (c *baseClient) ActivateEngine(ctx context.Context, opts clitypes.EngineInitOptions, out clitypes.OutStream,
-	authConfig *types.AuthConfig, healthfn func(context.Context) error) error {
+	authConfig *types.AuthConfig, healthfn func(context.Context) error, runtimeMetadataDir string) error {
```

Comment:
Instead of adding another argument, can we add `runtimeMetadataDir` as a field on `clitypes.EngineInitOptions` ?